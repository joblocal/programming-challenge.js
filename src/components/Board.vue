<template>
  <div class="Board">
    <Field
      v-for="(field, id) in fields"
      :key="id"
      :id="id"
    />
  </div>
</template>

<script>
import Field from 'src/components/Field';

export default {
  components: {
    Field,
  },

  data: () => ({
    fields: [0, 1, 2, 3, 4, 5, 6, 7, 8],
  }),
};
</script>

<style scoped>
.Board {
  position: relative;

  width: 100%;
  height: 100%;
}
</style>
