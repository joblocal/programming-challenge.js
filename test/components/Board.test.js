import Vue from 'vue';

// module under test
import Board from 'src/components/Board';

test('matches snapshot', () => {
  const vm = new Vue(Board).$mount();
  expect(vm.$el).toMatchSnapshot();
});
