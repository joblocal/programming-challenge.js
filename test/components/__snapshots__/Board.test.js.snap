// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  class="Board"
>
  <div
    class="Field"
  />
  <div
    class="Field"
  />
  <div
    class="Field"
  />
  <div
    class="Field"
  />
  <div
    class="Field"
  />
  <div
    class="Field"
  />
  <div
    class="Field"
  />
  <div
    class="Field"
  />
  <div
    class="Field"
  />
</div>
`;
